### DISCUSSION TITLE
Strange behavior when using join/alias

### DISCUSSION THREAD

**Discussion by calebeaires** [ClickHouse/ClickHouse#26122] [Q&A] [answered]
A simple query that makes use of join and set alias to is query brings the result with the name of the alias. The expectation is to bring the column name as it is on the database server.

Query
```sql
            SELECT
	              s.licenseId,
	              c.companyEmail
            FROM license s
	          LEFT JOIN company c ON c.companyId = s.companyId
```

Results

| s.licenseId  | c.companyEmail  |
|---|---|
| ffsdfgosogos  |  Some Name  | 
|  ogosodfosfs |  Enterprise |  

This **s.licenseId** and **c.companyEmail** should be **licenseId** and **companyEmail**

**Comment by vdimir** [comment, marked as answer]
Which version do you use? Also what is type of tables and schema (it should not matter, but maybe it's some unusual setup)?

On latest version it produces `licenseId` and `companyEmail`

```
SELECT 
    s.licenseId,
    c.companyEmail
FROM (SELECT 1 as companyId, 2 as licenseId) AS s
JOIN (SELECT 1 as companyId, 'example@example.com' as companyEmail) AS c
ON c.companyId = s.companyId;


┌─licenseId─┬─companyEmail────────┐
│         2 │ example@example.com │
└───────────┴─────────────────────┘
```
